package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/redhat-developer/yaml-language-server.git"
+    "url": "git+https://github.com/redhat-developer/yaml-language-server.git"
```

Comment:
I'm pretty sure this is just used for the link on the npm website. I think it should remain as-is instead of having `git+` at the beginning, because adding `git+` will break that link